src/main/resources/templates/mail/notification/sshKeyAddedEmail.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+
+<th:block th:replace="mail/notification/fragments :: head"/>
+
+<body>
+<!-- Message Body Start -->
+<th:block th:replace="mail/notification/fragments :: header" />
+
+<div id="message-body">
+    <th:block th:replace="mail/notification/fragments :: greetingsFullName" />
+
+    <p>
+    <span th:text="#{email.notification.sshKeyAdded.title}">The following SSH key was added to your account:</span>
+    </p>
+    <p>
+        <span th:utext="#{email.notification.sshKeyExpiry.sshKeyLabel}">SSH key label</span>
+        <span th:utext="${sshKey.getLabel()}">SSH key label</span>
+    </p>
+    <p>
+        <span th:utext="#{email.notification.sshKeyExpiry.sshKeyHash}">SSH key hash</span>
+        <span th:utext="${sshKey.getKeyHash()}">SSH key hash</span>
+    </p>
+    <p></p>
+    <p>
+        <span th:text="#{email.notification.sshKeyAdded.ifMistake}">If adding it was by mistake, remove it here:</span>
```

Comment:
```suggestion
        <span th:text="#{email.notification.sshKeyAdded.ifMistake}">If adding it was by mistake, you can remove it here:</span>
```

Reply:
Should we also mention something like: "If you did not add the ssh key, please report this to your administrator" and provide the Artemis instance email contact?